test/helpers/kubectl.go

```diff
@@ -395,47 +395,42 @@ func (kub *Kubectl) Logs(namespace string, pod string) *CmdRes {
 // a timeout. Also it returns a callback function to stop the monitor and save
 // the output to `helpers.monitorLogFileName` file.
 func (kub *Kubectl) MicroscopeStart() (error, func() error) {
-	/*
-		TODO: re-enable microscope in CI. See GH-4011.
-		microscope := "microscope"
-		cmd := fmt.Sprintf("%[1]s -n %[2]s exec %[3]s -- %[3]s",
-			KubectlCmd, KubeSystemNamespace, microscope)
-		_ = kub.Apply(microscopeManifest)
-
-		_, err := kub.WaitforPods(
-			KubeSystemNamespace,
-			fmt.Sprintf("-l k8s-app=%s", microscope),
-			300)
-		if err != nil {
-			return err, nil
-		}
-
-		ctx, cancel := context.WithCancel(context.Background())
-		res := kub.ExecContext(ctx, cmd, ExecOptions{SkipLog: true})
+	microscope := "microscope"
+	cmd := fmt.Sprintf("%[1]s -ti -n %[2]s exec %[3]s -- %[3]s",
+		KubectlCmd, KubeSystemNamespace, microscope)
+	_ = kub.Apply(microscopeManifest)
```

Comment:
Just `kub.Apply(...)`? `_` variable isn't needed if you don't want to get any value.

Reply:
@mrostecki in the CI code, we have hit unintentional bugs by ignoring return values of functions. I have requested in the past that we always specify that we are ignoring the result of a function with `_ =` so that it is clear that the writer of the test intended to ignore the result, and is not unintentionally doing so.

Reply:
OK, thanks for explanation.